beacon-chain/forkchoice/doubly-linked-tree/epbs.go

```diff
@@ -1,43 +1,38 @@
 package doublylinkedtree
 
 import (
-	"time"
-
 	"github.com/prysmaticlabs/prysm/v5/consensus-types/interfaces"
 	"github.com/prysmaticlabs/prysm/v5/consensus-types/primitives"
 	"github.com/prysmaticlabs/prysm/v5/time/slots"
 )
 
-func (n *Node) isParentFull() bool {
+func (n *BlockNode) isParentFull() bool {
 	// Finalized checkpoint is considered full
-	if n.parent == nil || n.parent.parent == nil {
+	if n.parent == nil {
 		return true
 	}
-	return n.parent.payloadHash != [32]byte{}
+	return n.parent.full
 }
 
 func (f *ForkChoice) GetPTCVote() primitives.PTCStatus {
 	highestNode := f.store.highestReceivedNode
 	if highestNode == nil {
 		return primitives.PAYLOAD_ABSENT
 	}
-	if slots.CurrentSlot(f.store.genesisTime) > highestNode.slot {
-		return primitives.PAYLOAD_ABSENT
-	}
-	if highestNode.payloadHash == [32]byte{} {
+	if slots.CurrentSlot(f.store.genesisTime) > highestNode.block.slot {
 		return primitives.PAYLOAD_ABSENT
 	}
-	if highestNode.withheld {
-		return primitives.PAYLOAD_WITHHELD
+	if highestNode.full {
+		return primitives.PAYLOAD_PRESENT
 	}
-	return primitives.PAYLOAD_PRESENT
+	return primitives.PAYLOAD_ABSENT
```

Comment:
Shouldn't this be `PAYLOAD_WITHHELD`? if not, where is withheld...

Reply:
I removed withheld in this iteration, adjusting for the minimal Francesco's forkchoice.